scripts/lldbinit.py

```diff
@@ -110,27 +111,23 @@ def print_dump(context: str, expr: str) -> None:
 
     # Try to find a type + id from the input args. If not, the id will be passed
     # through directly to C++, as it can be a variable name.
-    id_type = None
+    found_id_type = False
 
     # Look for <type><id> as a single argument.
-    if m := re.fullmatch("([a-z_]+)(\\d+)", args[1]):
+    if m := re.fullmatch("([a-z_]+)([0-9A-Fa-f]+)", args[1]):
         if m[1] in id_types:
             if len(args) != 2:
                 print_usage()
                 return
             id_type = m[1]
-            print_dump(context, f"{id_types[id_type]}({m[2]})")
-
-    # Look for <type> <id> as two arguments.
-    if not id_type:
-        if args[1] in id_types:
-            if len(args) != 3:
-                print_usage()
-                return
-            id_type = args[1]
-            print_dump(context, f"{id_types[id_type]}({args[2]})")
-
-    if not id_type:
+            if re.fullmatch("\\d+", m[2]):
+                id = m[2]
+            else:
+                id = int("0x" + m[2], 16)
```

Comment:
```suggestion
                id = int(m[2], 16)
```

I think specifying the base `16` makes the `0x` moot

```
>>> int("de", 16)
222
```

Reply:
Done